pyprof/nvtx/nvmarker.py

```diff
@@ -393,10 +397,80 @@ def patch_apex():
         import fused_adam_cuda
         patchClass(fused_adam_cuda)
 
+    if importlib.util.find_spec("fused_lamb_cuda") is not None:
+        import fused_lamb_cuda
+        patchClass(fused_lamb_cuda)
+
     if importlib.util.find_spec("fused_layer_norm_cuda") is not None:
         import fused_layer_norm_cuda
         patchClass(fused_layer_norm_cuda)
 
+    if importlib.util.find_spec("distributed_lamb_cuda") is not None:
+        import distributed_lamb_cuda
+        patchClass(distributed_lamb_cuda)
+
+    if importlib.util.find_spec("xentropy_cuda") is not None:
+        import xentropy_cuda
+        patchClass(xentropy_cuda)
+
+    if importlib.util.find_spec("mlp_cuda") is not None:
+        import mlp_cuda
+        patchClass(mlp_cuda)
+
+
+def patch_apex_pyt():
```

Comment:
Can we add a docstring to this? 
Something like patching all apex pytorch methods or whatever this is actually doing?